.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,73 @@
+name: Chromatic
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+    paths:
+      - frontend-react/**
+    branches-ignore:
+      - deployment
+
+jobs:
+  chromatic-deployment:
+    name: Chromatic
+    runs-on: ubuntu-latest
+    steps:
+      - name: Find Comment
+        id: comment_find
+        uses: peter-evans/find-comment@v2
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-regex: '\.*Chromatic.*\gi'
+
+      - name: Create comment
+        id: comment_create
+        uses: peter-evans/create-or-update-comment@v3
+        with:
+          comment-id: ${{ steps.comment_find.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            Deploying branch Storybook to Chromatic...
+          edit-mode: replace
+          reactions: rocket
+          reactions-edit-mode: replace
+
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # need commit history for Chromatic
+
+      - uses: actions/setup-node@v3.6.0
+        with:
+          node-version-file: frontend-react/package.json
+
+      - name: Install dependencies
+        run: yarn
+        working-directory: frontend-react
+        shell: bash
+
+      - name: Publish to Chromatic
+        id: chromatic_publish
+        uses: chromaui/action@v1
```

Comment:
current sub-version appears to be 6.17.3
`chromaui/action@d51b84e79d164fbe8fc5bb7175695d88ddd04b72`